src/redux/utils.ts

```diff
@@ -20,8 +20,7 @@ export const simpleRejectedActionToRemoteData = <ActionType extends string, M, E
 ): RemoteData.RemoteData<ApiError, never> =>
     RemoteData.failure(
         action.payload ?? {
-            code: ErrorCode.Unknown,
```

Comment:
@ingara Hvorfor ble ikke dette fanga av typescript?

Reply:
Hmm, godt spørsmål. Virker som at TS sin type-inferring er litt for lax akkurat her. Dersom vi spesifiserer `RemoteData.failure<ApiError>` så klarer den å catche det 🤔